src/pages/projects/projects-list/project-tile/ProjectTile.tsx

```diff
@@ -14,10 +15,8 @@ const ProjectTile = (props: Props) => {
         { props.project.name }
       </Link>
       <div className={styles.secondaryInfoContainer}>
-        <label> Created date: { props.project.createdDate } </label>
-        <label> Last modified: { props.project.lastModifiedDate } </label>
-        <label> contributors: { props.project.contributorsCount } </label>
-        <label> owner: { props.project.owner } </label>
+        <label>{props.project.owner}</label>
+        <label>Last modified: <TimeAgo live={false} date={props.project.modified}/></label>
```

Comment:
this is from epoch?

Reply:
Yup, https://github.com/nmn/react-timeago#date-required